frontend/src/components/OrganizationUpdate.vue

```diff
@@ -0,0 +1,36 @@
+<!--suppress XmlInvalidId -->
```

Comment:
Remove the ability to update organizations, as it is out of scope for now.

Reply:
done!